src/app/pages/bitcoin-contract-request/components/bitcoin-contract-offer/bitcoin-contract-offer-input.tsx

```diff
@@ -0,0 +1,56 @@
+import { useCallback } from 'react';
+
+import { Box, Text } from '@stacks/ui';
+import { truncateMiddle } from '@stacks/ui-utils';
+
+import { createMoneyFromDecimal } from '@shared/models/money.model';
+
+import { baseCurrencyAmountInQuote } from '@app/common/money/calculate-money';
+import { i18nFormatCurrency } from '@app/common/money/format-money';
+import { satToBtc } from '@app/common/money/unit-conversion';
+import { useCryptoCurrencyMarketData } from '@app/query/common/market-data/market-data.hooks';
+
+import { BitcoinContractLockAmount } from './bitcoin-contract-lock-amount';
+import { SimplifiedBitcoinContract } from '../../bitcoin-contract-request';
+
+interface BitcoinContractOfferInputProps {
+  addressNativeSegwit: string;
+  bitcoinContractOffer: SimplifiedBitcoinContract;
+}
+export function BitcoinContractOfferInput({
+  addressNativeSegwit,
+  bitcoinContractOffer,
+}: BitcoinContractOfferInputProps) {
+  const btcMarketData = useCryptoCurrencyMarketData('BTC');
+
+  const getFiatValue = useCallback(
```

Comment:
I believe I turned this into a utility recently, `useCalculateBitcoinFiatValue`.

Reply:
Replaced the useCallback with useCalculateBitcoinFiatValue, but when I give e.g. 0.0001 BTC, it returns a USD amount 264, not 2.64, what am I doing wrong? Is there any additional argument I need to give in somewhere? Thank you!

Reply:
The `Money` type we use expects values defined as the smallest currency unit, maybe that's why?

Reply:
Note here, `useCalculateBitcoinFiatValue` has been refactored to accept a `Money` type:
https://github.com/hirosystems/wallet/pull/3906